zerver/webhooks/groove/tests.py

```diff
@@ -0,0 +1,124 @@
+# -*- coding: utf-8 -*-
+from typing import Text
+
+from zerver.lib.test_classes import WebhookTestCase
+
+class GrooveHookTests(WebhookTestCase):
+    STREAM_NAME = 'groove'
+    URL_TEMPLATE = '/api/v1/external/groove?&api_key={api_key}'
+
+    # This test simulates the notification when
+    # there's a new ticket
+    def test_groove_ticket_started(self) -> None:
+        expected_subject = u"notifications"
+        expected_message = u"New ticket from Test Name\n" \
+                           u"```quote\n" \
+                           u"**[Ticket #9: Test Subject](https://ghostfox.groovehq.com/groove_client/tickets/68659446)**\n" \
+                           u"The content of the body goes here.\n" \
+                           u"```"
+        self.send_and_test_stream_message('ticket_started', expected_subject, expected_message,
+                                          content_type="application/x-www-form-urlencoded",
+                                          X_GROOVE_EVENT="ticket_started")
+
+    # This test simulates the notification when
+    # a ticket is assigned to an agent
+    def test_groove_ticket_assigned(self) -> None:
+        expected_subject = u"notifications"
+        expected_message = u"An open ticket has been assigned to agent@example.com\n" \
+                           u"```quote\n" \
+                           u"**[Ticket #9: Test Subject](https://testteam.groovehq.com/groove_client/tickets/68659446)**\n" \
+                           u"```"
```

Comment:
You can create prettier multiline strings by using brackets () instead of slashes \, or triple quotes """ instead.